extensions-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessAction.scala

```diff
@@ -70,23 +70,22 @@ object ScenarioActionName {
   val UnArchive: ScenarioActionName = ScenarioActionName("UNARCHIVE")
   val Pause: ScenarioActionName     = ScenarioActionName("PAUSE") // TODO: To implement in future..
   val Rename: ScenarioActionName    = ScenarioActionName("RENAME")
+  // TODO: We kept the old name of "run now" CustomAction for compatibility reasons.
+  //       In the future it can be changed to better name, according to convention, but that would require database migration
+  //       In the meantime, there are methods serialize and deserialize, which operate on name PERFORM_SINGLE_EXECUTION instead.
+  val RunNow: ScenarioActionName = ScenarioActionName("run now")
 
   val DefaultActions: List[ScenarioActionName] = Nil
 
   val StateActions: Set[ScenarioActionName] = Set(Cancel, Deploy, Pause)
 
-  // TODO: We kept the old name of "run now" CustomAction for compatibility reasons.
-  //       In the future it can be changed to better name, according to convention, but that would require database migration
-  //       In the meantime, there are methods serialize and deserialize, which operate on name PERFORM_SINGLE_EXECUTION instead.
-  val PerformSingleExecution: ScenarioActionName = ScenarioActionName("run now")
-
   def serialize(name: ScenarioActionName): String = name match {
-    case ScenarioActionName.PerformSingleExecution => "PERFORM_SINGLE_EXECUTION"
-    case other                                     => other.value
+    case ScenarioActionName.RunNow => "PERFORM_SINGLE_EXECUTION"
```

Comment:
Discussed outside PR:
- the legacy name is "run now" - used in db, displayed on frontend etc.
- the name in code was PerformSingleExecution, can be changed, but the serialized name should be consistent with it, it is used in FE API, so should be modified on FE too